From 17f9a92b27d0f2190b14d2728df2ce45d513417f Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <contact@chrisaw.io>
Date: Tue, 29 Sep 2026 16:09:53 +0100
Subject: [PATCH 1/1] drop the virtualization apex from the aosp mainline modules
 list

the virtualization apex is not built on the gsi, so the mainline
modules aggregate cannot require it.

Change-Id: I7f13a4efbdab6d5a515353314ed5273974884481
---
 build/Android.bp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build/Android.bp b/build/Android.bp
index 5f2b9a6..5065cfd 100644
--- a/build/Android.bp
+++ b/build/Android.bp
@@ -58,7 +58,6 @@ phony {
         "com.android.tethering",
         "com.android.tzdata",
         "com.android.uwb",
-        "com.android.virt",
         "com.android.wifi",
     ] + select(product_variable("use_debug_art"), {
         true: [
-- 
2.55.0

